src/styles/partials/template/_sidebar.scss

```diff
@@ -1,4 +1,5 @@
 .sidebar {
```

Comment:
I'm not sure why this is like this (i probably did it) but i suspect this should be `dg_sidebar`

Reply:
updated to dg_sidebar